src/therapy/schemas.py

```diff
@@ -104,7 +104,7 @@ class HasIndication(BaseModel):
     disease_id: str
     disease_label: str
     normalized_disease_id: Optional[str] = None
-    supplemental_info: Optional[Dict[str, str]] = None
+    supplemental_info: Optional[Dict[str, Optional[str]]] = None
```

Comment:
Was this supposed to be in the logging branch?

Reply:
no :(